fast64_internal/render_settings.py

```diff
@@ -14,8 +15,73 @@ def on_update_sm64_render_settings(self, context: bpy.types.Context):
 
 
 def on_update_oot_render_settings(self, context: bpy.types.Context):
-    # TODO: Update render properties from selected OOTLightProperty
-    pass
+    renderSettings: "Fast64RenderSettings_Properties" = context.scene.fast64.renderSettings
+    if renderSettings.ootSceneObject is None or not renderSettings.useObjectRenderPreview:
+        return
+    header = ootGetSceneOrRoomHeader(
+        renderSettings.ootSceneObject,
+        renderSettings.ootSceneHeader,
+        False,
+    )
+    if header is None:
+        return
+    lMode = header.skyboxLighting
+    if lMode == "0x01" or (lMode == "Custom" and not renderSettings.ootForceTimeOfDay):
+        if renderSettings.ootLightIdx >= len(header.lightList):
+            return
+        l = header.lightList[renderSettings.ootLightIdx]
+        renderSettings.ambientColor = tuple(c for c in l.ambient)
+        col0, dir0 = ootGetBaseOrCustomLight(l, 0, False, False)
+        renderSettings.lightColor = tuple(c for c in col0)
+        renderSettings.lightDirection = tuple(d for d in dir0)
+        # col1, dir1 = ootGetBaseOrCustomLight(l, 1, False, False)
+        # renderSettings.light1Color = tuple(c for c in col1)
+        # renderSettings.light1Direction = tuple(d for d in dir1)

```

Comment:
might be good to mark these as a todo so it doesnt look like dead code

```suggestion
        # TODO: Implement light1 into shader nodes
        # col1, dir1 = ootGetBaseOrCustomLight(l, 1, False, False)
        # renderSettings.light1Color = tuple(c for c in col1)
        # renderSettings.light1Direction = tuple(d for d in dir1)
```